tutorials/physics/interpolation/physics_interpolation_quick_start_guide.rst

```diff
@@ -0,0 +1,14 @@
+.. _doc_physics_interpolation_quick_start_guide:
+
+Quick start guide
+=================
+
+- Turn on physics interpolation: :ref:`ProjectSettings.physics/common/physics_interpolation<class_ProjectSettings_property_physics/common/physics_interpolation>`
```

Comment:
See the other comment about project settings for suggestions on how to style these links